### ISSUE TITLE
min-scale does not take effect immediately when tied to a variable.

### ISSUE THREAD

**Issue by jmikh** [timtnleeProject/vuejs-clipper#58]
when min-scale prop is tied to a variable, and that variable changes to something that is larger than the current zoom level, the min-scale does not get enforced immediately. It only gets enforced when I actually try to change zoom level using the scroll.

In the example GIF, the 4:3 button sets the min-scale to 1.333. You should expect the image get zoomed in immediately but it doesn't.

![ezgif-3-8e3f77b63b62](https://user-images.githubusercontent.com/4601618/79672960-3d393380-81a4-11ea-8b29-4944fe0ac696.gif)

**Comment by timtnleeProject** [comment]
When ratio change, the component will reset the scaling.
```javascript
resetData: function () {
  this.setTL$.next({ left: 0, top: 0 })
  const scale = (this.ratio > this.imgRatio)
    ? this.imgRatio / this.ratio
    : 1
  // origin --->
  this.setWH$.next(scale)
  // will be update in next version --->
  this.setWH$.next(Math.max(scale, this.minScale))
}
```

Also, be aware that the component will adjust the initial scaling by image ratio, so be careful of setting the `min-scale` too large.
![image](https://user-images.githubusercontent.com/33143314/79679124-b339ab00-8235-11ea-9d67-1bdc99302da4.png)

**Comment by timtnleeProject** [comment]
Fixed: v1.1.8
